src/Symfony/Component/Security/Http/Firewall/ContextListener.php

```diff
@@ -61,6 +63,7 @@ public function __construct(TokenStorageInterface $tokenStorage, $userProviders,
         $this->logger = $logger;
         $this->dispatcher = $dispatcher;
         $this->trustResolver = $trustResolver ?: new AuthenticationTrustResolver(AnonymousToken::class, RememberMeToken::class);
+        $this->rememberMeServices = $rememberMeServices;
```

Comment:
Should we throw a deprecation when this is not set?

Reply:
I guess no, since we have one per context just like @chalasr said ;).